src/frontend/components/Button/ButtonLayout/index.tsx

```diff
@@ -77,7 +83,9 @@ export const ButtonLayout = ({
       {Icon && (
         <IconBox screenSize={size}>
           <Icon iconColor={iconColor} focusColor={rectColor} height="100%" />
-          {badge}
+          <BadgeBox direction="row" align="start">
```

Comment:
can you use a stack from grommet ?

Reply:
I used what was already in the pre-existing component

Reply:
done in commit [3946e3f](https://github.com/openfun/marsha/pull/1443/commits/3946e3f7ca23f5295b8dafb02b70d3fd09039595)